views/jira-configuration.hbs

```diff
@@ -1,348 +1,254 @@
-            <div class='jiraConfiguration__syncRetryModalContent'>
-              <div class="jiraConfiguration__syncRetryModal__headerContainer">
-                <h4 class="jiraConfiguration__syncRetryModal__header">Backfill status</h4>
-                <span id="status-close" class="jiraConfiguration__syncRetryModal__closeBtn aui-icon aui-iconfont-close-dialog"></span>
-              </div>
-
-              <table class="jiraConfiguration__syncRetryModal__table">
-                <tbody class="jiraConfiguration__syncRetryModal__tableBody">
-                  <tr class="jiraConfiguration__syncRetryModal__tableRow">
-                    <td class="jiraConfiguration__syncRetryModal__syncStatus__type">
-                      <span class="syncStatusInProgress">In progress</span>
-                    </td>
-                    <td class="jiraConfiguration__syncRetryModal__syncStatus__details">
-                      Backfilling data has started and is still in progress for this account. New data may not immediately be displayed in Jira.
-                    </td>
-                  </tr>
-
-                  <tr class="jiraConfiguration__syncRetryModal__tableRow">
-                    <td class="jiraConfiguration__syncRetryModal__syncStatus__type">
-                      <span class="syncStatusFailed">Failed</span>
-                    </td>
-                    <td class="jiraConfiguration__syncRetryModal__syncStatus__details">
-                      There was a problem backfilling data from your account. The backfill process will automatically retry.
-                      However, if this does not happen, click the retry button next to the status, or "restart backfill" in the dropdown menu.
-                    </td>
-                  </tr>
-
-                  <tr class="jiraConfiguration__syncRetryModal__tableRow">
-                    <td class="jiraConfiguration__syncRetryModal__syncStatus__type">
-                      <span class="syncStatusPending">Pending</span>
-                    </td>
-                    <td class="jiraConfiguration__syncRetryModal__syncStatus__details">
-                      The backfill has been queued, but is not actively backfilling data from GitHub.
-                    </td>
-                  </tr>
-
-                  <tr class="jiraConfiguration__syncRetryModal__tableRow">
-                    <td class="jiraConfiguration__syncRetryModal__syncStatus__type">
-                       <span class="syncStatusFinished">Finished</span>
-                    </td>
-                    <td class="jiraConfiguration__syncRetryModal__syncStatus__details">
-                      The backfill has finished. Information from selected repositories will be shown in Jira’s development information panel.
-                    </td>
-                  </tr>
-                </tbody>
-              </table>
+  />
+  <script src="/public/js/jquery.min.js" nonce="{{nonce}}"></script>
+</head>
+
+<body class="aui-page-hybrid">
```

Comment:
Same here, quite some changes in the html only because of whitespaces. I guess if you can revert the changes to the whitespaces from main branch, these change will be disappear.
![image](https://user-images.githubusercontent.com/105693507/176830445-845932cc-e689-4ab8-8c0e-e7fdbae2d382.png)